x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_instances.tsx

```diff
@@ -38,7 +38,6 @@ export const alertInstancesTableColumns = (
       { defaultMessage: 'Instance' }
     ),
     sortable: false,
-    truncateText: true,
```

Comment:
I don't know if we're ok with removing this.
@mdefazio - any advice on whether we're ok with removing the truncate?
I assume this would cause warping of the table if names are too long?

Reply:
Yes, this will cause the text to wrap in the cell. This is probably a bit odd for something like a URL—but I can see the struggle to make sure the last bit of the URL is shown. Perhaps keep the truncate (I believe you also have to remove the `tableLayout=auto` for it to work), but define a width so it's wide enough for most cases like you have above.

Reply:
i have reverted the truncation part.

Reply:
I was reviewing this quickly and am perhaps a bit confused as to what you're trying to achieve.  It looks like the `tableLayout=auto` is still in there which disables the truncation if I'm not mistaken.

Reply:
@mdefazio just trying to make sure, url can be displayed :)

I wasn't sure what is the suitable width of instance column, instead i have added width for other columns which is predictable, is this an acceptable thing here? This was instance can take all the space and will truncate if it's not enough

![image](https://user-images.githubusercontent.com/3505601/97864311-b0516400-1d08-11eb-90fb-982ea77bd86b.png)

Reply:
I'll just add that it's important to see the full instance info, as-is it's not really usable. It's challenging when what's unique is long.

Reply:
Some options:

- display truncated, but provide full value in tooltip
- do the truncation in the "middle", so you'd see the beginning and end of the instance id, which I'm guessing is the rando stuff at the end of the image ^^^ - not sure if we have a "do truncation in the middle of the string" option, but I've seen this used fairly often in other sorts of tools.  For some reason I thought css was going to be adding this (from YEARS ago) as an additional [`text-overflow` property value](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow), but I guess I was dreaming.
- kinda punt on this, by providing a "navigateToApp" config, so instead of having to diagnose from here, you could go to the monitoring app to see the app-centric view of the situation, via a button click on the page.

Reply:
i think we can add tooltip while giving maximum space to instance column before truncating, that will resolve most concerns here.